README.md

```diff
@@ -99,20 +134,47 @@ Manage traditional [variants] of a configuration - like
 _development_, _staging_ and _production_ - using
 [overlays] that modify a common [base].
 
-![overlay image][imageOverlay]
+```
+
+overlay: kustomization + patches + more resources
+
+kustomization.yaml                              replica_count.yaml                     cpu_count.yaml
```

Comment:
Let's keep the patches themselves minimal and not repeat the selector and template meta sections from the base

Reply:
@KnVerey , done. Thanks.